---
title: "Results and Discussion & Conlcusion"
output: pdf_document
date: "`r Sys.Date()`"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library("annotate")
library("hgu133plus2.db")
library("readr")
raw.df = read.csv("../data/GSE28379.csv")
genes <- select(hgu133plus2.db, c(raw.df[,1]), c("SYMBOL","ENTREZID", "GENENAME"))
na.list <- genes[is.na(genes$SYMBOL),] # Storing the genes which were not detected for whatever reason.
colnames(genes)[1] <- "ID_REF" # Renaming  the column so the following merge works.
main <- merge(raw.df, genes, by=c("ID_REF")) # Merging the two dataframes together into a new one.
main <- main[!(main$ID_REF == "!series_matrix_table_end"),] # Removing an indicator row.
main <- main[order(main$log.2.fold.change),] # Reordering the DF by log2FC
main$path <- mapIds(hgu133plus2.db, c(main[,11]), c("PATH"), keytype = "ENTREZID")
main$go <- mapIds(hgu133plus2.db, c(main[,11]), c("ONTOLOGY"), keytype = "ENTREZID")
high <- main[pnorm(main$log.2.fold.change, mean = mean(main$log.2.fold.change),
                   sd = sd(main$log.2.fold.change)) >= 0.975,]
low <- main[pnorm(main$log.2.fold.change, mean = mean(main$log.2.fold.change),
                  sd = sd(main$log.2.fold.change)) <= 0.025,]
high <- high[order(high$log.2.fold.change, decreasing  = TRUE),]
low <- low[order(low$log.2.fold.change),]
```

## Results
Following the conclusion of exploratory data analysis, multiple important 
observations can be made about the data set. These will be examined by order
of importance to answering the research question.

Starting with the significant fold change values.

```{r echo=F}
head(high[,c(8,10)], 10)
head(low[,c(8,10)], 10)
```
These are the 10 most significant genes, for both the higher and lower ends. There
is a total of 57182 probe reads in the initial dataset. Using the pnorm function
within R, the genes with Log2FC values on both ends were extracted. The higher
end consisting of the upper 2.5% of the distribution, which totals 1761 of the 
57182 genes. The lower 2.5% consists of 1945 genes. These two numbers are close,
reinforcing the conclusion made in  the EDA that the data is normally distributed.
The higher the value, the more upregulated the gene is in the mutant samples.
The lower values are therefore indicators of downregulation in the mutant samples.

\newpage
A comparison between the two groups may be made. The following box plots display
summaries of both ends.

```{r echo=F}
boxplot(high$log.2.fold.change, main = "Log2FC Upregulated values")
boxplot(low$log.2.fold.change, main = "Log2FC Downregulated values")
```
The boxplot shows the usual summary values. It is noteworthy that there are
extreme values in both instances. 6 Of the upregulated values being higher
than the others in the plot. The same can be seen in the downregulation plot,
but with 7 values.

\newpage

```{r echo=F, fig.height=4}
d_high <- density(high$log.2.fold.change)
d_low <- density(low$log.2.fold.change)
plot(d_high, main="Density of Upregulation", xlab = "Log2FC")
plot(d_low, main="Density of Downregulation", xlab = "Log2FC")
```

Density plots also indicate that the downregulation is on average of bigger effect
than upregulation. 

\newpage
The results show above can also be demonstrated with a
numerical summary

```{r echo=F}
print(summary(high$log.2.fold.change))
print(summary(low$log.2.fold.change))
```
As can be seen, the downregulation has a more extreme mean than that of the
upregulation.

\newpage

In addition to the individual genes, ontological classes
were analysed. The following spread in ontological classes was observed.

```{r echo=F}
BP <- subset(main, go == "BP")
CC <- subset(main, go == "CC")
MF <- subset(main, go == "MF")

boxplot(BP$log.2.fold.change, CC$log.2.fold.change, MF$log.2.fold.change,
        names = c("BP", "CC", "MF"), ylab = "Log2FC", main = "Log2FC by GO class.")
```

This shows that the Log2FC spread differs for the classes. They do retain roughtly
the same averages, but have different lower and higher ends.

\newpage
This may also be observed for the significant DEGs.

```{r echo=F, fig.height=4}
BP_high <- subset(high, go == "BP")
CC_high <- subset(high, go == "CC")
MF_high <- subset(high, go == "MF")

BP_low <- subset(low, go == "BP")
CC_low <- subset(low, go == "CC")
MF_low <- subset(low, go == "MF")

boxplot(BP_high$log.2.fold.change, CC_high$log.2.fold.change, MF_high$log.2.fold.change,
        names = c("BP", "CC", "MF"), ylab = "Log2FC", main = "Log2FC in top 2.5% by GO class.")

boxplot(BP_low$log.2.fold.change, CC_low$log.2.fold.change, MF_low$log.2.fold.change,
        names = c("BP", "CC", "MF"), ylab = "Log2FC", main = "Log2FC in bottom 2.5% by GO class.")
```


\newpage

## Discussion & Conclusion
The examination of the data and plots shows that the mutation indeed affects
several genes. Both in being up and down regulated, with a slight bias towards
downregulation. In answering the research question, there needs to be
a correlation between genes and the development of FAD. This correlation seems
to be present, considering the mutation levels across the three mutation samples
are similar. This may further be researched with addition of other control groups,
and more FAD patient samples.

Which genes are correlated to FAD can also be deduced from the data. There's hundreds
of genes which are expressed significantly different in the mutant samples.
While all of these are important, there's still extremes with an alpha value of
0.05. These extreme downregulated genes are, in order of significance:
RPS4Y1, DDX3Y, EIF1AY, ZNF257, EIF1AY, HRK and TXLNGY. For upregulation these are:
BMP5, CASP1, MMP1, ITGB6, LINC02478, CD69.

### Upregulation
Looking at the individual genes and their transcripts may point to proteins
which would make sense for FAD patients to be functioning differently.
Looking at RPS4Y1: a gene encoding for  ribosomal protein S4. This specific gene,
and more related to ribosome components, have been observed to be differentially expressed in
multiple forms of Alzheimer's disease, not merely familial Alzheimer's disease. 
[Translational gene mapping of cognitive decline](https://www.sciencedirect.com/science/article/pii/S0197458006004349?via%3Dihub)

There's more relevant DEGs which affect the transcriptional capabilities of patients. These have
also been discovered in prior papers. DDX3Y is related to translation.
EIF1AY and ZNF257 are both genes encoding for transcription components, 
just like RPS4Y1.[Identification of potential blood biomarkers for early diagnosis of Alzheimer’s disease through RNA sequencing analysis](https://alzres.biomedcentral.com/articles/10.1186/s13195-020-00654-x)

The two remaining upregulated genes are neither related to the central dogma of cells.
HRK encodes for a gene related to activation and inhibition of apoptosis. This has also been observed
in a prior paper on AD genetics as a possibly important gene for AD. The hippocampus being mainly affected.
[Genomic Variants, Genes, and Pathways of Alzheimer’s Disease: An Overview](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6179157/)

TXLNGY is a gene detrimental to syntaxin binding activity. This gene appears to be
differentially expressed in AD patients who experience choroid plexus epithelium failure.
[Gene expression and functional annotation of human choroid plexus epithelium failure in Alzheimer’s disease](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4647590/)

### Downregulation
In the case of downregulation, the BMP5 gene has the lowest log2FC value. This gene
encodes for a protein which is part of developing cartilage and bones. The literature on
this specific gene is quite limited. Cartilage and bone development intuitively does not
allude to any involvement with development of Alzheimer's disease.

CASP1 is a gene that encodes for an enzyme which mediates the cleavage of the inactive precursor of IL-1beta.
The literature on the link between this and familial Alzheimer's disease is a topic of debate. This paper:  
[Caspase-1 genetic variation is not associated with Alzheimer's disease risk](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2837022/)
seems to acknowledge the differential expression correlation of this gene,
but does not accept the causative association with Alzheimer's disease.

MMP1 encodes for a metalloproteinase enzyme, which can cause tissue breakdown and secondary damage by several diseases.
According to this paper: [Brain matrix metalloproteinase 1 levels are elevated in Alzheimer's disease](https://pubmed.ncbi.nlm.nih.gov/10984641/), there might be a link with increased MMP1 levels and AD.
The keyword being elevated. This DEG data set indicates that the gene is down regulated in case of FAD. 
This is a surprising result, considering the referenced paper shows the opposite. 

The literature on the link between [ITGB6] and FAD is limited, but the differential expression of this gene
is a recurring observation. The protein for which the gene encodes is an integrin.

LINC02478 is a bit of a different gene to the others. This is a long intergenic non-coding rna gene. This might be cause for further analysis, because this gene lacks sufficient research to make any yet conclusions.

The last gene, CD69, encodes for a membrane-bound, type 2 C-lectin receptor.
The paper: [Elevation of CD69+ monocyte/macrophages in patients with Alzheimer's disease](https://pubmed.ncbi.nlm.nih.gov/10879686/)
Indicates that this is indeed a relevant DEG. But just like MMP1, there's a difference in up/down regultaion between the data
in the referenced paper, and the data set used in this one. While the reference shows
that there is a link between upregulation and AD, the data from this project shows
downregulation.

It is not clear why some of the genes in this project seem to have opposing regulation
trends with the data seen in other papers. This might be because both up and down
regulation of these genes are pathogenic, and may lead to the loss of function in proteins.
Which would explain the phenomenon.
