% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{article}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={Results and Discussion \& Conlcusion},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi

\title{Results and Discussion \& Conlcusion}
\author{}
\date{\vspace{-2.5em}2022-10-03}

\begin{document}
\maketitle

\hypertarget{results}{%
\subsection{Results}\label{results}}

Following the conclusion of exploratory data analysis, multiple
important observations can be made about the data set. These will be
examined by order of importance to answering the research question.

Starting with the significant fold change values.

\begin{verbatim}
##       log.2.fold.change    SYMBOL
## 15347          6.856410      BMP5
## 21720          6.657270     CASP1
## 14353          6.378964      MMP1
## 37774          5.905406     ITGB6
## 37775          5.905406 LINC02478
## 20067          5.554548      CD69
## 21719          5.175524     CASP1
## 332            4.736748    CARD16
## 333            4.736748     CASP1
## 21331          4.672425       HGF
\end{verbatim}

\begin{verbatim}
##       log.2.fold.change SYMBOL
## 11737         -9.370551 RPS4Y1
## 14897         -8.308893  DDX3Y
## 14288         -8.047485 EIF1AY
## 21020         -7.655585 ZNF257
## 14287         -7.256954 EIF1AY
## 40891         -6.912372    HRK
## 44064         -6.881689 TXLNGY
## 51508         -6.268827   <NA>
## 14898         -6.189748  DDX3Y
## 39797         -6.054623  USP9Y
\end{verbatim}

These are the 10 most significant genes, for both the higher and lower
ends. There is a total of 57182 probe reads in the initial dataset.
Using the pnorm function within R, the genes with Log2FC values on both
ends were extracted. The higher end consisting of the upper 2.5\% of the
distribution, which totals 1761 of the 57182 genes. The lower 2.5\%
consists of 1945 genes. These two numbers are close, reinforcing the
conclusion made in the EDA that the data is normally distributed. The
higher the value, the more up-regulated the gene is in the mutant
samples. The lower values are therefore indicators of down-regulation in
the mutant samples.

\newpage

A comparison between the two groups may be made. The following box plots
display summaries of both ends.

\includegraphics{Results-and-Discussion---Conclusion_files/figure-latex/unnamed-chunk-2-1.pdf}
\includegraphics{Results-and-Discussion---Conclusion_files/figure-latex/unnamed-chunk-2-2.pdf}
The boxplot shows the usual summary values. It is noteworthy that there
are extreme values in both instances. 6 Of the up-regulated values being
higher than the others in the plot. The same can be seen in the
down-regulation plot, but with 7 values.

\includegraphics{Results-and-Discussion---Conclusion_files/figure-latex/unnamed-chunk-3-1.pdf}
\includegraphics{Results-and-Discussion---Conclusion_files/figure-latex/unnamed-chunk-3-2.pdf}

Density plots also indicate that the down-regulation is on average of
bigger effect than up-regulation. In addition to plotting, this can also
be demonstrated with a numerical summary

\begin{verbatim}
## [1] "test"
\end{verbatim}

\begin{verbatim}
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##   1.350   1.565   1.848   2.046   2.314   6.856
\end{verbatim}

\begin{verbatim}
## [1] "test"
\end{verbatim}

\begin{verbatim}
##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##  -9.371  -3.169  -2.723  -2.899  -2.402  -2.187
\end{verbatim}

As can be seen, the down-regulation has a more extreme mean than that of
the up-regulation.

\hypertarget{discussion-conclusion}{%
\subsection{Discussion \& Conclusion}\label{discussion-conclusion}}

The examination of the data and plots shows that the mutation indeed
affects several genes. Both in being up and down regulated, with a
slight bias towards down regulation. In answering the research question,
there needs to be a correlation between genes and the development of
FAD. This correlation seems to be present, considering the mutation
levels across the three mutation samples are similar. This may further
be researched with addition of other control groups, and more FAD
patient samples.

Which genes are correlated to FAD can also be deduced from the data.
There's hundreds of genes which are expressed significantly different in
the mutant samples. While all of these are important, there's still
extremes with an alpha value of 0.05. These extreme down-regulated genes
are, in order of significance: RPS4Y1, DDX3Y, EIF1AY, ZNF257, EIF1AY,
HRK and TXLNGY. For up-regulation these are: BMP5, CASP1, MMP1, ITGB6,
LINC02478, CD69.

Looking at the individual genes and their transcripts may point to
proteins which would make sense for FAD patients to be functioning
differently. Looking at RPS4Y1: a gene encoding for ribosomal protein
S4. This specific gene, and more related to ribosome components, have
been observed to be differentialy expressed in multiple forms of
Alzheimers, not just FAD.
\url{https://www.sciencedirect.com/science/article/pii/S0197458006004349?via\%3Dihub}
There's more DEG's which affect the transcriptional capabilities of
patients. These have also been discovered in prior studies. DDX3Y is
related to translatoin. EIF1AY and ZNF257 are both genes encoding for
transcription components, just like RPS4Y1. (another paper on
transcriptional componontent mutation in AD):
\url{https://alzres.biomedcentral.com/articles/10.1186/s13195-020-00654-x}

The two other up-regulated genes are neither related to the cental dogma
of cells. HRK encodes for a gene related to activation and inhibition of
apoptosis. This has also been observed in prior papers on AD genetics as
a possibly important gene for AD ( Hippocampus ).
(\url{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6179157/})

TXLNGY seems to be a gene detrimental to syntaxin binding activity. This
gene seems to be differentially expressed in AD patients with choroid
plexus epithelium failure.
\url{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4647590/}

In the case of down regulation, the BMP5 gene has the lowest log2FC
value. This gene encodes for a protein which is part of developing
cartilige and bones. The literature on this specific gene is quite
limited. Cartilige and bone development intuitively does not allude to
any involvement with development of Alzhimer's disease.

CASP1 is a gene that encodes for an enzyme which mediates the cleavage
of the inactive precursor of IL-1Î². The literature on the link between
this and familial alzheimers disease is a topic of debate. This paper
(\url{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2837022/}) seems to
acknowledge the differential expression of this gene, but does not
accept the association with Alzheimer's disease.

\end{document}
