---
title: "Exploratory Data Analysis: Which genes and their expression levels may be associated with developing familial alzheimer's disease?"
author: "Orfeas Gkourlias"
date: "`r Sys.Date()`"
output:
  pdf_document: default
  html_document:
    df_print: paged
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
#Bioconductor packages
library("annotate")
library("hgu133plus2.db")
library("readr")
library("tidyr")
```

\newpage
# Codebook
```{r}
codebook <- read_delim(file = "codebook.txt",
                       delim = ";",
                       show_col_types=FALSE)

knitr::kable(codebook)
```
\newpage

\section{Introduction}
The aim of this project is to find links between certain gene expressions and familial Alzheimer's disease, using machine learning. To be more specific for the sake of data analysis: The mutation being observed is a presenilin 2 mutation, using patient-specific induced pluripotent stem cells (iPSC) to facilitate expression of the mutant type. Four different expression profiles were collected, using the Affymetrix Human Genome U133 Plus 2.0 Array. When looking at the names of columns, genes and differing values of expression, it's important to consider those are are all Affymetrix standards, which may need to be converted to further down the line. For example: Converting the gene IDs to ensembl IDs.
\newpage   

\section{Initial Data and Variables}
Let's first take a look at the provided .csv file, it's structure and first entries.

```{r}
raw.df = read.csv("../data/GSE28379.csv")
head(raw.df, 5)
```
## ID_Ref.
This column indicates the probe ID's, as sequenced by the  Affymetrix Human Genome U133 Plus 2.0 Array. This is athe result of the sequencing technique. These are probe ID's, which don't represent a lot by themselves. They can, however, be used to find the ensembl ID's and gene symbols, which will be attempted below with Bioconductor:

```{r}
genes <- select(hgu133plus2.db, c(raw.df[,1]), c("SYMBOL","ENTREZID", "GENENAME"))
na.list <- genes[is.na(genes$SYMBOL),] # Storing the genes which were not detected for whatever reason.
colnames(genes)[1] <- "ID_REF" # Renaming  the column so the following merge works.
main <- merge(raw.df, genes, by=c("ID_REF")) # Merging the two dataframes together into a new one.
main <- main[!(main$ID_REF == "!series_matrix_table_end"),] # Removing an indicator row.
main$path <- mapIds(hgu133plus2.db, c(main[,11]), c("PATH"), keytype = "ENTREZID")
main$go <- mapIds(hgu133plus2.db, c(main[,11]), c("ONTOLOGY"), keytype = "ENTREZID")
head(main[,c(1,10,11,12)], 5)
```
To summarize what has just been done: A bioconductor database was used to
find the corresponding gene for every probe. It's important to note that for yet unknown reasons
some probes were not recognized. The proper database was used, as can be seen by the database name.

## GSM
The GSM columns indicate the different samples used in the paper this data is derived from.
The values under these columns represent sequencing concentration, which is the result of a normalisation algorithm called MAS5.0. This algorithm is also developed by Affymetrix. These values are also not log transformed. A dedicated column was made for that.

The first two samples, GSM701542 and GSM701543 are iPSC sequences derived from Sparadic Parkinson's disease patients. The latter two, 
GSM701544 and GSM701545 are iPSC sequences from familial Alzheimer's disease (FAD) patients. The referenced paper aimed to compare
the two conditions and their gene expressions.

## No mutation & Mutation
The first column, no mutation, signifies the average of the first two non mutated parkinson's samples.
The second column, mutation, shows the average of the two FAD mutant type samples.

## Log2 Fold Change
These are subtracted 2log fold change values, showing which of the two averages are up regulated and down regulated.
In case of a positive number, the mutation type samples are up regulated and the non-mutant types are down regulated.
The reverse is true in case of a negative number.

## Fold change
The ratio between the mutation and no mutation values. Mutation being divided by no mutation in this case.
\newpage 

# Data variance & spread.
The original paper aimed to compare two groups and their expressions. By looking at the calculated Log2FC data, it'll be possible to see how much the two groups differ. Let's first look at the most significant differently expressing genes.

```{r}
main <- main[order(main$log.2.fold.change),] # Reordering the DF by log2FC
head(main[,c(10,6,7,8)], 10) #Showing the 10 most significant down regulated genes
tail(main[,c(10,6,7,8)], 10) #Showing the 10 most significant up regulated genes
```
\newpage
By taking a glance at the tables above, it seems that down regulation consists of more extreme values than up regulation. While this may be telling of how expression is affected by the mutation in general, it's not enough on its own to draw any conclusions yet. Let's further explore the log2fc values by creating a boxplot.     

```{r}
boxplot(main$log.2.fold.change, horizontal = TRUE, xlab = "Log2FC", title = "Boxplot of Log2FC")
```

As can be seen from the boxplot, there appears to be great variance in the log2FC values.
This may be further examined by looking at a summary.
```{r}
summary(main$log.2.fold.change)
```
\newpage

Judging by the box plot and summary table, the genes seem to be mostly down regulated.

To further look at the distribution of the  data, a density plot can also be used.

```{r, fig.width=8, fig.height=5}
plot(density(main$log.2.fold.change), main = "Log2FC distribution", xlab = "Log2FC")
```
Judging by the box plots and density plot, the data seems to be roughly normally distributed.
This is useful when looking at statistical significance in analyzing correlations.
The lower and higher tail end genes will be stored for these ends.

```{r}
high <- main[pnorm(main$log.2.fold.change, mean = mean(main$log.2.fold.change),
                   sd = sd(main$log.2.fold.change)) >= 0.975,]
low <- main[pnorm(main$log.2.fold.change, mean = mean(main$log.2.fold.change),
                  sd = sd(main$log.2.fold.change)) <= 0.025,]
```

\newpage

# Correlations
Since the data in the file are all individual reads, there will not be any variables
that affect each other, besides the obvious calculation of fold changes and means.
That being said: There could be a connection between up/down regulation of certain
genetic regions. Taking RPS4Y1, a gene encoding for a ribosomal sub unit,
as an example: Could there be a trend in the mutation affecting down/up regulation
of specifically ribosomal genes? Questions like these are not suited for EDA, but 
the addition of gene symbols, KEGG pathway, go and entrezID's columns in this document 
will help in answering such questions, which will eventually mean answering the research question.

Plotting the different KEGG IDs and ontological classifications can however offer insight into
which pathways and classes are most prevalent.

```{r}
barplot(table(unlist(main$go)), xlab = "Ontological Class", ylab = "Occurences")
```
The differing classes appear to be close in the amount of occurrences. Looking at
the values in the lower and upper 2.5%  may net different results.

\newpage

```{r}
barplot(table(unlist(high$go)), xlab = "Ontological Class", ylab = "Occurences", main ="Upper-tail")
barplot(table(unlist(low$go)), xlab = "Ontological Class", ylab = "Occurences", main ="Lower-tail")
```
The bar plots show that the lower-end and upper-end tails are also equally divided.

\newpage

It may be possible to find distinctions between different classes by looking at 
the Log2FC values for every different class.

```{r}
BP <- subset(main, go == "BP")
CC <- subset(main, go == "CC")
MF <- subset(main, go == "MF")

summary(BP$log.2.fold.change)
summary(CC$log.2.fold.change)
summary(MF$log.2.fold.change)

boxplot(BP$log.2.fold.change, CC$log.2.fold.change, MF$log.2.fold.change,
        names = c("BP", "CC", "MF"), ylab = "Log2FC")
```
Both the summaries and box plots are very similar for every classification.
At first glance this indicates that the classes are not a detriment in
the fold change values of genes.

To confirm this suspicion, t-test may be performed. Since multiple
groups are being observed, an ANOVA test may be used.
```{r}
main[is.na(main$go),]$go <- "unknown"
main[main$go == "NULL",]$go <- "unknown"
summary(aov(main$log.2.fold.change~unlist(main$go)))
```
As can be seen from the results, the P-Value indicates that there's a significant
difference in Log2FC means of ontological classes.

The same may be done for the pathways. The pathway column does have significantly more
unique occurrences than the ontological classifications. The ones with the most significant
log2FC values will therefore be shown and considered.

```{r}
occur_path_high <- table(unlist(high$path))
occur_path_high <- occur_path_high[order(occur_path_high, decreasing = TRUE)]
occur_path_high[1:10] # list is around 160 diferent pathway entries long. 
#Showing the 10 most prevelant ones at the higher end.

occur_path_low <- table(unlist(low$path))
occur_path_low <- occur_path_low[order(occur_path_low, decreasing = TRUE)]
occur_path_low[1:10] # list is around 160 diferent pathway entries long. 
#Showing the 10 most prevelant ones at the lower end.
```
\newpage

# Summary
The initial data offers insight into the differing expression values of genes between
a mutant and non-mutant group. By examining the variables and columns provided,
multiple facets of the data was analysed. First, there seems to be a great spread in the data
when looking at log2FC values. The data is also normally distributed. 
To find correlations between the variables, notably log2FC and different identifiers,
the most significant DEGs were extracted. Analysis of these genes and their complimentary
KEGG pathways and ontological identifiers was performed. No significant correlations were
found by looking at those two variables and examining the corresponding log2FC values.